contracts/backend.go

```diff
@@ -0,0 +1,60 @@
+package contracts
+
+import (
+	"math/big"
+
+	"github.com/celo-org/celo-blockchain/common"
+	"github.com/celo-org/celo-blockchain/core/vm"
+	"github.com/celo-org/celo-blockchain/params"
+)
+
+type evmBackend struct {
+	vmConfig    vm.Config
+	vmContext   vm.Context
+	chainConfig *params.ChainConfig
+	state       vm.StateDB
+
+	dontMeterGas bool
+}
+
+func NewEVMBackend(vmContext vm.Context, state vm.StateDB, chainConfig *params.ChainConfig, vmConfig vm.Config) Backend {
+	return &evmBackend{
+		vmConfig:    vmConfig,
+		vmContext:   vmContext,
+		chainConfig: chainConfig,
+		state:       state,
+	}
+}
+
+// Call implements Backend.Call
```

Comment:
Saying `evmBackend` implements `contracts/utils.go:Backend` and then documenting these functions would be more clear.